src/Makefile

```diff
@@ -1500,90 +1507,21 @@ EXTRA_CFLAGS += -fPIC
 RTOBJS := $(sort $(foreach mod,$(obj-m),$(call TORTOBJS,$(mod))))
 
 RTDEPS := $(sort $(patsubst $(OBJDIR)/%.o,$(DEPDIR)/%.d, $(RTOBJS)))
-
```

Comment:
On 3/11/2016 10:41 AM, Michael Brown wrote:
> 
> And learn how to re base a commit properly.

Quick and dirty method, since there's not a lot of complex changes (so
wading through learning git rebase is probably overkill):

* Grab the latest machinekit/master

* Checkout a new branch based on master

* Update the new branch with changes, by cherry-pick(ing) commits from
your previous working branch and/or copying over the new/changed files

* Commit and sign-off

* Send a PR to machinekit repo

-- 
Charles Steinkuehler
charles@steinkuehler.net